features/non-english-hyphens.yml

```diff
@@ -0,0 +1,67 @@
+name: Non-English hyphens
+description: The `hyphens` CSS property uses language-specific rules to determine when long words are broken by line wrapping. The content language is set with the HTML `lang` attribute.
```

Comment:
I feel like this description should make it clearer that this is for non-English and that readers are supposed to have read the description for `hyphens` first. 
I know it's in the name, but I feel like there needs to be more of a connection between the two features somehow.

Having a `see_also` field would be useful for this kind of feature. Or, perhaps, we can put them together in a new group, maybe called `typography` (which would be under the CSS group).

As for the description, how about something like this: "When used with a non-English HTML `lang` attribute value, the `hyphens` CSS property uses language-specific rules to determine when long words are broken by line wrapping."

Reply:
> Having a `see_also` field would be useful for this kind of feature.

I agree, that would be useful. 

For the description, I want to stay away from the idea that English is the default- per the spec, you should need to also specify your language as English in order to get English hyphens.  As I think through this, the reason I split them apart was to tell a fuller support story, but unless we split each language into a feature, it won't be possible for someone to differentiate between French support (baseline low) and Esperanto support (none). 

Because of this, I'm proposing that I combine the two into a single `hyphens` feature, and adding a note that "support for non-English languages varies". While that violates the "don't mention support in the description" rule from the style guide, it does seem important to let people know. Thoughts?

Reply:
I've joined them in https://github.com/web-platform-dx/web-features/pull/1714/commits/e730703740535f1f76e51077955eb7e4f308b5d6